### ISSUE TITLE
Mapping doesn't line up with the bed dimensions

### ISSUE THREAD

**Issue by DickyDck** [npiegdon/bed-inspector#18]
Hello and thanks for helping me on the youtube page! I more or less got it working. I am using Windows 10 x64, connected to an Anycubic Kobra Standard. Leaving the autodetect selected for Marlin 1.x and Marlin 2.x , matched BAUD rate on my laptop to match what the default is in the BedLevel Program. It connects, moves the printer to home and does a single drop marker on the screen. However when I go to Measure corners, the printer goes to each corner, but doesn't mark anything on the screen. When I click on the screen it goes to that location and drops the level sensor down, and marks it on the screen, but if I move the mouse just an inch or so away it moves the printer head, but on the screen it drops a mark across the screen totally far away. Sometimes it behaves in the opposite behavior when I go to a far off location, it will mark a dot practically on top of the dot it was just on. I am sure I am doing something wrong and it isn't the software, any assistance would be greatly appreciated.

**Comment by npiegdon** [comment]
This sounds a little like #17, at least in the "homing works but everything else behaves strangely" sense.  This is one of those things that's tricky to debug without having the 3D printer in question.

Have you noticed any difference in the log output between that first marker that is drawn correctly vs. any subsequent points that aren't?  Is the output format changing, even slightly?  (Is it possible to work around those changes by entering a custom string on the Detection tab?)

**Comment by DickyDck** [comment]
> This sounds a little like #17, at least in the "homing works but everything else behaves strangely" sense. This is one of those things that's tricky to debug without having the 3D printer in question.
> 
> Have you noticed any difference in the log output between that first marker that is drawn correctly vs. any subsequent points that aren't? Is the output format changing, even slightly? (Is it possible to work around those changes by entering a custom string on the Detection tab?)

I'll try the custom string on the detection tab today after work, do I need to uncheck the two Marlin boxes when using Custom?

**Comment by npiegdon** [comment]
> ... do I need to uncheck the two Marlin boxes when using Custom?

No.  It'll check any/all of the three whenever a new log line comes in from the printer.  It's safe to leave the others checked.

**Comment by DickyDck** [comment]
ok tried that and actually made things worse LOL, I'll try a few other setting changes and see what happens.

**Comment by npiegdon** [comment]
What sort of output are you seeing after each time it tries to measure a point?

**Comment by DickyDck** [comment]
when switching to custom, it goes to one point then freezes, like hard freeze and then I have to close the app and start over. Something about using custom has angered the bed deity :p

**Comment by vmachacek** [comment]
i have similar issues, when connected to Ender-3 S1 i see two points come up and than the procedure stops. It looks like that is mismatch with the regex which is not found, I getting messages in this format

```
X:110.00 Y:110.00 Z:12.90 E:0.00 Count X:8800 Y:8800 Z:5160
``` 

@npiegdon if you can provide guidance what this might be i can make a PR

**Comment by vmachacek** [comment]
also when i print out everything from  `DataReceived` i get this

```
ok
ok
ok
ok
echo:busy: processing
echo:busy: processing
echo:busy: processing
echo:busy: processing
echo:busy: processing
echo:busy: processing
echo:busy: processing
X:141.80 Y:150.50 Z:10.00 E:0.00 Count X:11344 Y:12040 Z:4000
echo:Bed Leveling ON
echo:Fade Height 10.00
echo:Bed Leveling ON
echo:Fade Height 10.00
ok
ok
ok
echo:busy: processing
echo:busy: processing
echo:busy: processing
echo:busy: processing
echo:busy: processing
Bed X: 78.20 Y: 69.50 Z: -0.05
X:110.00 Y:110.00 Z:12.90 E:0.00 Count X:8800 Y:8800 Z:5160
ok
ok
ok
echo:busy: processing
echo:busy: processing
echo:busy: processing
echo:busy: processing
echo:busy: processing
Bed X: 48.20 Y: 29.50 Z: -0.18
X:80.00 Y:70.00 Z:12.90 E:0.00 Count X:6400 Y:5600 Z:5160
ok
ok
ok
ok
```

and than it stops

**Comment by npiegdon** [comment]
Hmm, strange.  I think I've seen output with both kinds of lines before (the kind that starts with "Bed X:..." and the kind that starts with "X:...").  The worrying part is that the two never seem to agree.  In your output it looks like the X's and Y's travel the same distance between both kinds of report, but the Z's are 12.90 & -0.05 at first and then 12.90 & -0.18 later.  The identical 12.90 is worrying.

On the detection tab, what happens if you uncheck both Marlin options and enable custom with the following: "X:{X} Y:{Y} Z:{Z}"?  That should make it listen for the other kind of report.  (I just noticed that field doesn't allow copy or paste; sorry!)
